frontend/challedu-app/src/APIUtils.js

```diff
@@ -19,6 +19,7 @@ export default class APIUtils {
 }
 
 async function issueAJAXRequest(method, endpoint, body = {}, access_token = "") {
+  console.log("acces"+access_token);
```

Comment:
Please commit this file without the console.log()

Reply:
@zannis 
F.Y.I.
Paulin specifically asked me, when I created that function, to include that console.log.